apps/api/src/app/auth/auth.controller.ts

```diff
@@ -8,6 +8,8 @@ import {
 import { AuthGuard } from "@nestjs/passport"
 import { Response, Request } from "express"
 
+const REDIRECT_URL = "http://localhost:4200/my-groups"
```

Comment:
This should be dynamic, bcs those values change based on our environment. So we should use the env variables here.